README.md

```diff
@@ -56,6 +56,9 @@ To register a device, run as root from LmP device:
  $ DEVICE_API=http://<IP of docker-compose host>/sign lmp-device-register [-T <TOKEN>] [-f <factory>]
 ~~~
 
+**NOTE** If you're emulating the device with QEMU, by default the gateway (host) IP is `10.0.2.2`.
+(i.e. `sudo DEVICE_API=http://10.0.2.2:80/sign lmp-device-register [-T <TOKEN>] [-f <factory>]`)
```

Comment:
Qemu is surely picking a random/free IP when you run. This probably isn't repeatable across everyone's systems